src/test/java/io/debezium/connector/vitess/VitessConnectorIT.java

```diff
@@ -229,7 +229,7 @@ public void shouldSameTransactionLastRowOffsetBeNewVgtid() throws Exception {
                 // last row event has the new vgtid
                 assertRecordOffset(record, RecordOffset.fromSourceInfo(record));
             }
-            assertSourceInfo(record, TestHelper.TEST_SERVER, table.schema(), table.table());
+            assertSourceInfo(record, TestHelper.TEST_UNSHARDED_KEYSPACE, table.schema(), table.table());
```

Comment:
`assertSourceInfo` is for Vitess specifically, could we refactor it to verify all the fields that's meaningful for us? e.g., 
```
protected void assertSourceInfo(SourceRecord record, String db, String keyspace, String schema, String table)
```

Reply:
Done refactoring this. Added name and rename parameter to keyspace.